### PR TITLE
docs: add debug common errors guide 

### PR DESCRIPTION
**Pull request by deboragaleano**
This is the "How to debug common development errors" guide linked to _Set up your development environment_ tutorial - [PR 7](https://github.com/dhis2/developer-portal/pull/7). 

Reference: Developer Portal content updates [document](https://docs.google.com/document/d/1-QfPhG80_EVSN1X6z4CTjYeTo2fbTEt3tgNNnPPvBjo/edit?usp=sharing).

### CODE REVIEW

**Review by amcgee — APPROVED** [review]
I've sent the academy version of this to several people (including our newest developer @karolinelien) and it has been SUPER helpful!

**Comment by amcgee** [comment]
@chathurawidanage actually ran into another issue that it might be good to add to this guide - if you use the extension [HTTPs Everywhere](https://www.eff.org/https-everywhere) from EFF in any browser (Chrome, Firefox) and set the `Encrypt All Sites Eligible` setting it will break when using (unencrypted) http://localhost:3000 -- the extension tries to use `https://localhost:3000` for CORS;


https://user-images.githubusercontent.com/947888/113126053-8cdb0880-9217-11eb-8d83-24fdee53ddd5.mov

**Comment by deboragaleano** [comment]
@amcgee that's great! Thanks so much, I'll add the HTTPs Everywhere extension issue as well 👍

File: docs/guides/debug-instance.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+You could also try disabling the cache from the network tab in Chrome DevTools. See below:
+
+![](./assets/disable-cache.png)
+
+## Using DHIS2 Play instances
+
+**DHIS2 Play instances**: [https://play.dhis2.org/](https://play.dhis2.org/) allow you to test out the DHIS2 platform using demo databases on the DHIS2 Play server. 
+
+If you try to connect your application or localhost to the DHIS2 Play server and encounter some errors, please note that DHIS2 Play instances use [nginx](https://nginx.org/) as reverse proxy and this configuration is designed for security reasons, preventing cross-site cookies completely.
```
Comment (amcgee) [inline docs/guides/debug-instance.md:50]:
```suggestion
If you try to connect your application on localhost to the DHIS2 Play server and encounter some errors, please note that DHIS2 Play instances use [nginx](https://nginx.org/) as a reverse proxy which has been configured for hardened security, preventing cross-site cookies completely. 
```

File: docs/guides/debug-instance.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+![](./assets/disable-cache.png)
+
+## Using DHIS2 Play instances
+
+**DHIS2 Play instances**: [https://play.dhis2.org/](https://play.dhis2.org/) allow you to test out the DHIS2 platform using demo databases on the DHIS2 Play server. 
+
+If you try to connect your application or localhost to the DHIS2 Play server and encounter some errors, please note that DHIS2 Play instances use [nginx](https://nginx.org/) as reverse proxy and this configuration is designed for security reasons, preventing cross-site cookies completely. 
+
+To work around this issue during application development is to run a local instance of DHIS2 using Docker and the `d2 cluster` command of the [DHIS2 CLI](https://cli.dhis2.nu/#/commands/d2-cluster). Check [this guide](./spin-up-local-instance) on how to spin up a local instance.
```
Comment (amcgee) [inline docs/guides/debug-instance.md:52]:
```suggestion
To work around this issue during application development, run your own development instance of DHIS2 or spin up a local instance using Docker and the `d2 cluster` command of the [DHIS2 CLI](https://cli.dhis2.nu/#/commands/d2-cluster). Check [this guide](./spin-up-local-instance) on how to spin up a local instance. 
```

**Review by amcgee — APPROVED** [review]
Some small suggestions but looks great as well!

**Comment by deboragaleano** [comment]
@thanks Austin! 🙂

> Some small suggestions but looks great as well!

**Pull request merged**